clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -2271,6 +2298,18 @@ private StoreSetting getSettingsFromController(ControllerClient controllerClient
     storeSetting.isWriteComputeEnabled = storeResponse.getStore().isWriteComputationEnabled();
     storeSetting.isIncrementalPushEnabled = storeResponse.getStore().isIncrementalPushEnabled();
     storeSetting.storeRewindTimeInSeconds = DEFAULT_RE_PUSH_REWIND_IN_SECONDS_OVERRIDE;
+    if (setting.singleRegionPush && setting.targetedRegions == null) {
```

Comment:
should we fail if both set ? or could we make the logic be  must enable feature flag, then optionally set the list of target regions else it will use the NR source fabric ?

Reply:
> could we make the logic be must enable feature flag, then optionally set the list of target regions else it will use the NR source fabric ?

I think this is a great idea. Will do it